From a86645062f55ffc148525c270262addc2c59f3d5 Mon Sep 17 00:00:00 2001
From: Jin Xue <jin.xue@intel.com>
Date: Fri, 14 Dec 2018 15:05:36 +0800
Subject: [PATCH 4/6] Revert "Revert "primary audio hal" to enable KOG."

This reverts commit a4a79c34511f6dce858cf29cc2588cc28a11c9b8.

Change-Id: Ie3ebcfa86ac9aeaa285c70c6ef8af2985c8e48d1
Reviewed-on: 655448
Reviewed-by: Zhang, Hongyu <hongyu.zhang@intel.com>
Tested-by: Zhang, Hongyu <hongyu.zhang@intel.com>
Reviewed-on: 657228
---
 Android.mk | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/Android.mk b/Android.mk
index 68c26bf..ccf2807 100644
--- a/Android.mk
+++ b/Android.mk
@@ -12,8 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ifeq ($(INTEL_AUDIO_HAL),audio)
-
 LOCAL_PATH := $(call my-dir)
 
 include $(CLEAR_VARS)
@@ -35,15 +33,10 @@ LOCAL_C_INCLUDES += \
 	$(call include-path-for, audio-route) \
 	$(call include-path-for, audio-effects)
 
-#Preferred paths for all vendor hals /vendor/lib/hw
-LOCAL_PROPRIETARY_MODULE := true
-
-LOCAL_MODULE := audio.primary.$(TARGET_BOARD_PLATFORM)
+LOCAL_MODULE := audio.primary.$(TARGET_PRODUCT)
 
 LOCAL_MODULE_RELATIVE_PATH := hw
 
 LOCAL_MODULE_TAGS := optional
 
 include $(BUILD_SHARED_LIBRARY)
-
-endif # INTEL_AUDIO_HAL
-- 
2.7.4

